app/build.gradle

```diff
@@ -10,7 +10,7 @@ android {
         minSdkVersion 17
         targetSdkVersion 25
         versionCode 1
-        versionName "1.0"
+        versionName getVersionName()
```

Comment:
odkial sa berie toto version name?

Reply:
ak to suvisi s Kotlinom, ok, ale typicky sa pouziva system ako je popisany tu:
https://plus.google.com/+JakeWharton/posts/6f5TcVPRZij
Nie je to samozrejme povinne, ale clovek si moze sam navolit verziu, zvysovanie minor verzie atd.